tests/backup/backup_test.go

```diff
@@ -371,6 +371,9 @@ var _ = Describe("{BackupCrashVolDriver}", func() {
 			ValidateApplications(contexts)
 		})
 
+		// Wait for IO to run
```

Comment:
Just a thought, since the same time.Sleep is used in BackupKillStork test we can make it configurable

Reply:
Done